man/chpl.rst

```diff
@@ -145,8 +145,7 @@ OPTIONS
 
     Enables the cache for remote data. This cache can improve communication
     performance for some programs by adding aggregation, write behind, and
-    read ahead. This cache is not enabled by any other optimization
-    *options* such as **--fast**.
+    read ahead.
```

Comment:
Should it say that `--cache-remote` is the default?

Reply:
I don't think we do that with any other optimizations.